### ISSUE TITLE
Previous session flickering on external monitor

### ISSUE THREAD

**Issue by clinejj** [warpdotdev/Warp#818]
### Describe the bug

```text
When using Warp on a secondary monitor (Dell U2421HE), the previous session window is flickering. This does not occur on my main monitor (Macbook Pro M1 14"). This only occurs with previous session blocks, it does not occur with current session blocks.

Using version v0.2022.03.14.08.49.stable_00
```

### To Reproduce

Open Warp with a previous session, and move window to other monitor.

### Expected behaviour

```text
There should be no flickering.
```

### Screenshots

https://user-images.githubusercontent.com/2057878/159081561-5d2cbfd3-3aa6-4b9f-b8ac-f06cbb7ed52c.mp4

### Operating System

MacOS

### OS Version

12.3

**Comment by clinejj** [comment]
Update - I am not seeing this issue on a different computer/monitor (Macbook Air M1, with Dell U2415 also on MacOS 12.3)

**Comment by MauroLG** [comment]
I'm experiencing the same issue. It only happens with Warp and when I have a previous session, as mentioned on this issue. The external monitor's screen starts flickering at the edges and leaves a ghosting effect. I've tried different monitor settings and different cables, but the issue remains the same.
The only way it seems to not happen is if I set a much lower resolution than usual (< 1080), considering the monitor is 1440.
I've also experienced instances where the monitor flickers when opening a NEW Warp window, but it doesn't always happen.
Lately, I thought it might be something very specific and that the monitor has hardware problems, but I highly doubt it's a coincidence.

Hardware: MacBook Pro M1 16"; Dell u2715h 27"
Warp version: v0.2023.09.06.18.09.stable_00
